.github/workflows/lint.yml

```diff
@@ -39,3 +39,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: chartboost/ruff-action@v1
+
+  clang-format-lint:
+    runs-on: ubuntu-24.04
+    container:
+      image: "registry.opensuse.org/home/mia/images/images/mpv-ci:stable-deps"
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 50
+      - name: Configure git
+        # Workaround to avoid "fatal: detected dubious ownership in repository" error.
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
@llyyr: thanks for adding clang-tools, works fine. Do you think we could add `safe.directory` config in container directly? We have the same in build.yml, which is fine, but we could just add `*` as `safe.directory` in image to avoid those configs.

Reply:
Done